httpreplay/cobweb.py

```diff
@@ -2,15 +2,18 @@
 # This file is part of HTTPReplay - http://jbremer.org/httpreplay/
 # See the file 'LICENSE' for copying permission.
 
-import dpkt
+import binascii
 import logging
+import re
 import zlib
 
+import dpkt
 from httpreplay.exceptions import UnknownHttpEncoding
 from httpreplay.shoddy import Protocol
 
 log = logging.getLogger(__name__)
 
+
```

Comment:
No need for the 2-liner between each function / class. Yes, I know it violates pep8, but personal preference ;)